lib/src/Pos/Diffusion/Full/Block.hs

```diff
@@ -724,6 +724,7 @@ handleStreamStart logTrace logic oq = listenerConv logTrace oq $ \__ourVerInfo n
                       lift $ traceWith logTrace (Debug, sformat ("handleStreamStart:loop MsgStart, expected MsgStreamUpdate from "%build) nodeId)
                       return ()
                   MsgUpdate u -> do
+                      lift $ OQ.clearFailureOf oq nodeId
```

Comment:
This is already on develop!
https://github.com/input-output-hk/cardano-sl/blob/develop/lib/src/Pos/Diffusion/Full/Block.hs#L727